### ISSUE TITLE
[Bug]: `torch.compile(..., backend='openvino')` requires `import openvino.torch`

### ISSUE THREAD

**Issue by anzr299** [openvinotoolkit/openvino#27421]
### OpenVINO Version

2024.4.0

### Operating System

Ubuntu 20.04 (LTS)

### Device used for inference

CPU

### Framework

PyTorch

### Issue description

When trying to compile a model with openvino backend, it gives an error `AssertionError: duplicate name: openvino`. This seems to go away when importing `import openvino.torch`.
Torch version == 2.5.0

### Step-by-step reproduction

Here is a minimally reproducible code which gives the error:
```
import torch
class SimpleTestModel(torch.nn.Module):
    def __init__(self):
        super().__init__()
    def forward(self, x):
        return torch.add(x,10)
compiled_model = torch.compile(SimpleTestModel(), backend='openvino')
```

### Issue submission checklist

- [X] I'm reporting an issue. It's not a question.
- [X] I checked the problem with the documentation, FAQ, open issues, Stack Overflow, etc., and have not found a solution.
- [X] There is reproducer code and related data files such as images, videos, models, etc.

**Comment by rkazants** [comment]
Hi @anzr299,

It should have been recently fixed in https://github.com/openvinotoolkit/openvino/pull/27245 that is going to 2024.5 release.
In order to check, please try our latest openvino nightly build:

```
pip install openvino-nightly -U
```

Best regards,
Roman

**Comment by anzr299** [comment]
> Hi @anzr299,
> 
> It should have been recently fixed in #27245 that is going to 2024.5 release. In order to check, please try our latest openvino nightly build:
> 
> ```
> pip install openvino-nightly -U
> ```
> 
> Best regards, Roman

This seems to fix the issue. Thank you for the response!
